scripts/update-crd-reference/main.sh

```diff
@@ -2,21 +2,15 @@
 
 set -e
 
-CRD_DOCS_GENERATOR_VERSION=0.7.1
+CRD_DOCS_GENERATOR_VERSION=0.8.0
 DESTINATION=src/content/ui-api/management-api/crd
 
 # Clear output folder
 find ${DESTINATION} -type f -not -name "_index.md" | xargs rm
 
-# Determine apiextensions version
-VERSION=$(docker run --rm --entrypoint /bin/sh quay.io/giantswarm/docs-scriptrunner -c "curl --silent https://api.github.com/repos/giantswarm/apiextensions/releases/latest|jq -r .tag_name|tr -d '\n'")
-
-echo "Version: ${VERSION}"
-
```

Comment:
We no longer get the latest version in this script. This is now hard coded in the config.

We can find ways in the future to keep that up-to-date.